rrfs-test/IODA/yaml/prepbufr_aircft.yaml

```diff
@@ -31,112 +29,110 @@ observations:
             query: "*/YOB"
           stationIdentification:
             query: "*/SID"
-          aircraftFlightNumber:
-            query: "[*/ACID, */ACID_SEQ/ACID]"
-          prepbufrDataLevelCategory:
-            query: "*/PRSLEVLA/CAT"
+          stationElevation:
+            query: "*/ELV"
+            type: float
           prepbufrReportType:
             query: "*/TYP"
           dumpReportType:
             query: "*/T29"
           prepbufrDataLvlCat:
-            query: "*/PRSLEVLA/CAT"
+            query: "*/PRSLEVLA{1}/CAT"
           aircraftPhase:
-            query: "*/PRSLEVLA/ACFT_SEQ/POAF"
-          stationElevation:
-            query: "*/ELV"
-            type: float
+            query: "*/PRSLEVLA{1}/ACFT_SEQ/POAF"
+          aircraftFlightNumber:
+            query: "[*/ACID, */ACID_SEQ/ACID]"
 
           pressure:
-            query: "*/PRSLEVLA/P___INFO/P__EVENT{1}/POB"
+            query: "*/PRSLEVLA{1}/P___INFO/P__EVENT{1}/POB"
             transforms:
               - scale: 100
           pressureQualityMarker:
-            query: "*/PRSLEVLA/P___INFO/P__EVENT{1}/PQM"
+            query: "*/PRSLEVLA{1}/P___INFO/P__EVENT{1}/PQM"
           pressureError:
-            query: "*/PRSLEVLA/P___INFO/P__BACKG/POE"
+            query: "*/PRSLEVLA{1}/P___INFO/P__BACKG/POE"
             transforms:
               - scale: 100
           pressureTunedError:
-            query: "*/PRSLEVLA/P___INFO/P__POSTP/POETU"
+            query: "*/PRSLEVLA{1}/P___INFO/P__POSTP/POETU"
             transforms:
               - scale: 100
 
           heightOfObservation:
-            query: "*/PRSLEVLA/Z___INFO/Z__EVENT{1}/ZOB"
+            query: "*/PRSLEVLA{1}/Z___INFO/Z__EVENT{1}/ZOB"
             type: float
```

Comment:
not sure about this, as the old yaml already has "float"?
@spanNOAA @HuiLiu-NOAA Could you double check this?